### PR TITLE
vendor: move to go dep tool

### PR DESCRIPTION
**Pull request by philips**

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/tfjson](https://github.com/palantir/tfjson), @philips! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

**Comment by dtolnay** [comment]
@jeremy-asher please figure out with Tyler the ownership of this repo. I don't have merge permission since I quit.

**Comment by philips** [comment]
@dtolnay @jeremy-asher I am happy to "adopt" the project if no one is actively maintaining it and move it to github.com/coreos.

**Comment by dtolnay** [comment]
That sounds great! @nmiyake can we make that happen?

**Comment by dtolnay** [comment]
I think it would be a matter of finding someone who has permission to create repos in the coreos org, adding them with admin permission on this repo, and then they can [transfer](https://help.github.com/articles/about-repository-transfers/) it.

**Comment by philips** [comment]
I fit that bill for Admin on coreos org.

**Comment by nmiyake** [comment]
Great -- I will need to check internally, but don't anticipate that there will be any issue with this. Will echo back when I have confirmation.

**Comment by nmiyake** [comment]
For the PR itself, is the `go dep` tool referenced https://github.com/golang/dep, or something else? I've been keen to check out the tool/evaluate moving projects to it, but have hesitated for production things since the project still notes that it is in alpha and the commands and file formats are subject to change. Curious to hear thoughts if you have any more info (or is this a different tool?).

**Comment by philips** [comment]
It is that tool. Essentially the entire import tree needed to be rewritten
so I used this tool. Happy to use a different tool too. What was the
project originally using?

**Comment by nmiyake** [comment]
Nope fine with that tool, was just curious as to whether it was at the maturity level where production projects were using it. Typically we use `govendor`, but honestly I'm not very happy with it and am eagerly anticipating moving to `dep`.

I'm fine with merging this PR while I wait to hear back about transferring the repo.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ version of [Terraform PR #3170](https://github.com/hashicorp/terraform/pull/3170
 ## Installation
 
 ```
-$ go get github.com/palantir/tfjson
+$ go get github.com/philips/tfjson
```
Comment (nmiyake) [inline README.md]: This change should be omitted since this is a PR into the `palantir/tfjson` repo.

Comment (philips) [inline README.md]: gah, oops. Sorry, PR is based on master. I will close this out in favor of: https://github.com/palantir/tfjson/pull/4

**Pull request closed without merging**